binding.cc

```diff
@@ -390,8 +390,7 @@ namespace zmq {
 
   void
   Socket::UV_PollCallback(uv_poll_t* handle, int status, int events) {
-    assert(status == 0);
-
+    if (status != 0) NanThrowRangeError("I/O status: socket not ready !=0 ");
```

Comment:
> range errors for when an argument value was outside some allowed range

hey what's the value to this argument, you think status is not an argument value?